trento/xml/article_sap_trento.xml

```diff
@@ -442,7 +440,11 @@ As agreed on https://confluence.suse.com/x/DAEcN on our Trento doc kick off
           Note that the experimental flag is not needed as of Helm version 3.8.0.
         </para>
       </step>
-      <step>
+       <step>
+        <para> Monitor the creation and startup of the Trento Kubernertes pods and wait until they are all in running status:</para>
```

Comment:
We have an entity for Kubernetes (to avoid typos): :wink: 

```suggestion
        <para> Monitor the creation and startup of the Trento &k8s; pods and wait until they are all in running status:</para>
```